starwars/src/components/StarWarsCard.js

```diff
@@ -1,27 +1,16 @@
 import React from "react"
-import {
-    Card,
-    CardText,
-    CardBody,
-    CardTitle,
-    CardSubtitle,
-    Row,
-} from "reactstrap"
+import { Card, CardText, CardBody, CardTitle, CardSubtitle } from "reactstrap"
 
 const StarWarsCard = props => {
     return (
-        <Card>
-            <Row>
-                <CardBody>
-                    <CardTitle>{props.name} </CardTitle>
-
-                    <CardSubtitle>Birthday: {props.birth}</CardSubtitle>
-
-                    <CardText>
-                        {props.name} has {props.hair} hair and {props.eye} eyes.
-                    </CardText>
-                </CardBody>
-            </Row>
+        <Card className='card'>
+            <CardBody className='main'>
+                <CardTitle className='top'>{props.name} </CardTitle>
+                <CardSubtitle>Birthday: {props.birth}</CardSubtitle>
+                <CardText>
+                    {props.name} has {props.hair} hair and {props.eye} eyes.
+                </CardText>
+            </CardBody>
         </Card>
```

Comment:
Very good use of reactstrap here. Forming the data into a sentence is a creative use of the data. Used the components properly and it is semantic and logical.